aiida/transports/transport.py

```diff
@@ -186,6 +187,16 @@ def get_valid_auth_params(cls):
 
     @classproperty
     def auth_options(cls):  # pylint: disable=no-self-argument
+        """Return the authentication options to be used for building the CLI.
+
+        :return: `OrderedDict` of tuples, with first element option name and second dictionary of kwargs
+        """
+        # The common auth options are currently defined as class members, but the default for `safe_interval` is sub
+        # class specific. With the current design the default cannot already be specified directly but has to be added
+        # manually here.
+        for option in cls._common_auth_options:
```

Comment:
hm, nasty...
how about making ` _common_auth_options` and `_valid_auth_options` `OrderedDict`s themselves. that would make accessing them easier

Reply:
That will make it impossible to set the correct default though, see the comment.

Reply:
hm... I don't quite follow. How does `_common_auth_options` being an `OrderedDict` instead of a list of tuples change this?
Anyhow it would just be a minor improvement, not important.